src/layout/FileUpload/FileUploadTable/FileTable.tsx

```diff
@@ -68,12 +67,28 @@ export function FileTable({
             className={classes.blueUnderline}
             id='altinn-file-list-row-header'
           >
-            <th>{lang('form_filler.file_uploader_list_header_name')}</th>
-            {!mobileView ? <th>{lang('form_filler.file_uploader_list_header_file_size')}</th> : null}
-            {hasTag ? <th>{lang(tagTitle)}</th> : null}
-            {!(hasTag && mobileView) ? <th>{lang('form_filler.file_uploader_list_header_status')}</th> : null}
             <th>
-              <p className='sr-only'>{lang('form_filler.file_uploader_list_header_delete_sr')}</p>
+              <Lang id={'form_filler.file_uploader_list_header_name'} />
+            </th>
+            {!mobileView ? (
```

Comment:
I agree that it looks nicer,  the only issue I've had with it is the potential for (falsy?) values to actually be displayed instead. But yeah, in this case it would be better.